update: updated notation key command based on spec (#416)
notaryproject/notation: specs/commandline/key.md

Add key to signing key list

Usage:
  notation key add --plugin <plugin_name> [flags] <key_name>

Flags:
  -d, --default                mark as default
  -h, --help                   help for add
      --id string              key id (required if --plugin is set)
  -p, --plugin string          signing plugin name

>       --plugin-config strings  {key}={value} pairs that are passed as is to a plugin, if the verification is associated with a verification plugin, refer plugin documentation to set appropriate values

Reviewer: `plugin-config` is passed for both signing and verification plugin. [ref](https://github.com/notaryproject/notaryproject/blob/main/specs/plugin-extensibility.md#generate-signature)
```suggestion
      --plugin-config strings  {key}={value} pairs passed as it is to a plugin, refer plugin's documentation to set appropriate values
```
